src/components/Product/ProductList/ProductList.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import ProductItem from "./ProductItem/ProductItem.jsx";
+import Modal from "../../Modal/Modal.jsx";
+import "./ProductList.css";
+import NoFoundProductImg from "../../../../public/img/no-product-found.png";
+
+export default function ProductList(props) {
+    const {
+        products,
+        searchedProductName,
+        cartProducts,
+        setCartProducts,
+        setIsCartShakingAnimation,
+    } = props;
+
+    const [isModalActive, setIsModalActive] = useState(false);
+    const [imageValue, setImageValue] = useState({ src: null, name: null, price: null });
+    const filterProductsItem = products.filter(product => product.name.toLowerCase().startsWith(searchedProductName.toLowerCase()));
+
+    function zoomProductImage (e) {
+        setIsModalActive(isModalActive => !isModalActive);
+
+        const parentNode = e.target.closest('.productItem');
+        let getProductId = parentNode.id;
+        let getProductInfo = products.find(product =>
+            product.id === parseInt(getProductId)
+        );
+
+        setImageValue({
+            src : e.target.src,
```

Comment:
Шо за лишні пробіли при свойствах обьекта? Просто: 
```suggestion
            src: e.target.src,
```